org.roojs.roobuilder.json

```diff
@@ -0,0 +1,114 @@
+{
+    "app-id": "org.roojs.roobuilder",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+    	"org.freedesktop.Sdk.Extension.vala"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path" : "/usr/lib/sdk/vala/lib"
+    },
+    "command": "roobuilder",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+    	"--socket=session-bus",
+    	"--filesystem=home",
+    	"--filesystem=/usr/share/vala-0.56",
+    	"--filesystem=/usr/share/vala",    	
```

Comment:
this won't work. `/usr` remains read only. Also you seem to want to assume this is availble on the host.